docs/production-install.rst

```diff
@@ -105,7 +97,7 @@ for more information on this file.
 
 You'll need to restart postgresql if the configuration was updated:
 ::
-  sudo service postgresql-9.3 restart
+  sudo systemctl restart postgresql-9.3
 
 Disable SELinux
```

Comment:
At some point we need to stop saying "Disable SELinux" and shave this yak.  SELinux isn't going anywhere. :black_joker: